hyperspy/tests/io/test_jeol.py

```diff
@@ -245,3 +272,141 @@ def test_shift_jis_encoding():
+        fname = str(dirname / filename)
+
+        # Count number of frames including incomplete frame
+        data = hs.load(fname, sum_frames = False, only_valid_data = False,
+                       downsample=[32,32], rebin_energy=512, SI_dtype=np.int32)
+        assert data.axes_manager["Frame"].size == frames
+   
+        # Count number of valid frames
+        data = hs.load(fname, sum_frames = False, only_valid_data = True,
+                       downsample=[32,32], rebin_energy=512, SI_dtype=np.int32)
+        assert data.axes_manager["Frame"].size == valid
+
+def test_em_image_in_pts():
+    dir1 = TESTS_FILE_PATH
+    dir1p = dir1 / 'Sample' / '00_View000'
+    dir2 = TESTS_FILE_PATH / 'InvalidFrame'
+    dir2p = dir2 / 'Sample' / '00_Dummy-Data'
+    
+    # no SEM/STEM image
+    s = hs.load(dir1 / test_files[0],
+                read_em_image=False, only_valid_data=False)
+    assert len(s) == 7
+
+    s = hs.load(dir1 / test_files[0],
+                read_em_image=True, only_valid_data=False)
+    assert len(s) == 7
+
+    # with SEM/STEM image
+    s = hs.load(dir2 / test_files2[0],
+                read_em_image=False, only_valid_data=False)
+    assert len(s) == 22
+    s = hs.load(dir2 / test_files2[0],
+                read_em_image=True, only_valid_data=False)
+    assert len(s) == 25
+    assert s[8].metadata.General.title == "S(T)EM Image extracted from " + s[8].metadata.General.original_filename
+    assert s[8].data[38,15] == 87
+    assert s[8].data[38,16] == 0
+
+    # integrate SEM/STEM image along frame axis
+    s = hs.load(dir2p / test_files2[16], read_em_image=True,
+                only_valid_data=False, sum_frames=True,
+                frame_list=[0,0,0,1])
+    assert(s[1].data[0,0] == 87*4)
+    assert(s[1].data[63,63] == 87*3)
+    
+    s = hs.load(dir2p / test_files2[16], read_em_image=True,
+                only_valid_data=False, sum_frames=False)
+    s2 = hs.load(dir2p / test_files2[16], read_em_image=True,
+                   only_valid_data=False, sum_frames=True)
+    s1 = [s[0].data.sum(axis=0), s[1].data.sum(axis=0)]
+    assert np.array_equal(s1[0], s2[0].data)
+    assert np.array_equal(s1[1], s2[1].data)
+
+def test_pts_lazy():
+    dir2 = TESTS_FILE_PATH / 'InvalidFrame'
+    dir2p = dir2 / 'Sample' / '00_Dummy-Data'
+    s = hs.load(dir2p / test_files2[16], read_em_image=True,
+                only_valid_data=False, sum_frames=False, lazy=True)
+    s1 = [s[0].data.sum(axis=0).compute(),
+          s[1].data.sum(axis=0).compute()]
+    s2 = hs.load(dir2p / test_files2[16], read_em_image=True,
+                only_valid_data=False, sum_frames=True, lazy=False)
+    assert np.array_equal(s1[0], s2[0].data)
+    assert np.array_equal(s1[1], s2[1].data)
+
+def test_pts_frame_shift():
+    dir = TESTS_FILE_PATH2
+    dir2p = dir / 'Sample' / '00_Dummy-Data'
+    file = str(dir2p / test_files2[16])
+    #    dir = TESTS_FILE_PATH
+    #    file = str(dir / 'testpattern1.pts')
```

Comment:
Is it worth testing these files? If not, they should be removed? If they need to be removed, it would be good to remove them from the history to avoid unnecessarily increasing the size of the git repository. If is a bit annoying to do, I can do ti if necessary.

Reply:
Ok, I agree to remove those files.
I'm sorry I'm not friendly with complicate GIT operation.
So, please do it on your side.